launch_testing/launch_testing/asserts/assert_output.py

```diff
@@ -64,6 +65,9 @@ def assertInStdout(proc_output,
     sequences from actual output before comparing with the output filter or
     expected output.
     :type strip_ansi_escape_sequences: bool
+
+    :param stream: Which stream to examine.  This must be one of 'stderr' or 'stdout'.
```

Comment:
@clalancette meta: it's not currently implemented, but what if one wants to check both streams' output, in chronological order? Maybe a `'both'` option may help that use case, which is the supported one by process proxies (see [here](https://github.com/ros2/launch/blob/b99eb3811b80fe9c685151543b5dbb690a52674b/launch_testing/launch_testing/tools/process.py#L64-L82)).

Reply:
It's certainly functionality that we can add, though I'd prefer not to add it until we run across a use-case that needs it.

Reply:
I'd generally agree with that, but the change is small and IMO it'll reduce the likelihood of users hacking their way forward if they ever want to do that. 

Having said that, I'm fine to move forward if you strongly prefer not to allow this patch to get bigger.

Reply:
Yeah, for now I'm going to err on the side of keeping this smaller (I already have to add one more thing here based on discussion from a couple of days ago).  If (probably when) a user opens an issue, it should be an easy addition.

Reply:
The problem with "chronological" is that in my experience, since we'd be implementing this by capturing both and interleaving them (AFAIK), the order of messages is not deterministic.